frontend/src/packages/chat-messages/libs/enums/enum.ts

```diff
@@ -0,0 +1 @@
+export { ChatsApiPath } from 'shared/build/index.js';
```

Comment:
frontend/src/packages/chat-messages/libs/enums/enums.ts

Reply:
Not resolved